src/Services/Capsules/HasCapsules.php

```diff
@@ -65,6 +81,16 @@ public function getCapsulesSubdir()
 
     public function makeCapsule($capsule, $basePath = null): array
     {
+        $capsule['composer'] = $capsule['composer'] ?? false;
+
+        if ($capsule['composer'] === true) {
+            if (class_exists(InstalledVersions::class) === false) {
```

Comment:
```suggestion
            if (class_exists('\Composer\InstalledVersions') === false) {
```
I'm not 100% sure, forgive me if I'm missing something. I'm wondering if the check on `InstalledVersions::class` actually makes sense if the class doesn't exist (ie. Composer < 2.0). Also, I think a check on `method_exists('\Composer\InstalledVersions', 'getInstallPath')` could also be needed to complete the check for Composer 2.1.

On a related note, should this check happen earlier in the process, perhaps in `getCapsuleList()`?